docs/add-non-um-users.html

```diff
@@ -3,5 +3,37 @@
 title: Add Non-UM Users
```

Comment:
```suggestion
title: Add Non U-M Users
```

Reply:
@melindakraft This is something talked about with @janelilr and UX team. This is how it should be shown